docs/admin/deployment.rst

```diff
@@ -106,6 +107,10 @@ you create:
    Optional. URL to the page displayed to your users when the application encounters
    a system error. See `Heroku Reference`_ for more information.
 
+``GITLAB_URL``
+   Optional. URL of your GitLab instace when ``AUTHENTICATION_METHOD=gitlab``.
+   If not defined, ``https://gitlab.com`` will be used by default.
+
```

Comment:
I'd leave that out as we don't have docs for other provider-specific variables.

Reply:
Hum,

* Google: does not seems to require any additional env var to work (only some config in the Django backoffice),

* Github: the allauth doc say that the URL can be configured to support Github Entreprise but this is not supported by the current implementation in Pontoon so there is no specific env var for this provider too.

* FXA: can be configured using env var, and... this is not documented... Maybe we should add some documentation for the FXA env vars instead?

Reply:
Oops, didn't saw the vars for google and github, I will look at this tomorrow  ^^'